### DISCUSSION TITLE
Selectively turn gpg on and off

### DISCUSSION THREAD

**Discussion by kaji-ryoji** [simple-login/app#662] [Feature Requests]
Currently if I turn gpg off for a single mailbox, I won't be able to turn on gpg for an individual alias.

However I'd like to see that gpg can be configured to be default off, but toggled on for an individual alias

**Comment by nguyenkims** [comment]
@kaji-ryoji I wonder if you can do the inverse, i.e. leave the PGP enabled on the mailbox and disable PGP on aliases where it isn't needed?
